src/resources/V2/Core/Events.ts

```diff
@@ -1,12 +1,59 @@
-// File generated from our OpenAPI spec
-
+// This file is manually maintained
 import {StripeResource} from '../../../StripeResource.js';
+
 const stripeMethod = StripeResource.method;
+const RelatedObjects = StripeResource.extend({
+  fetchRelatedObject(pulledEvent: any) {
```

Comment:
Why not add `fetchRelatedObject()` directly on `Events` which also extends from StripeResource instead of creating a whole new StripeResource?

Reply:
I wasn't able to 100% convince myself that there wasn't some state held in StripeResource that pointed to a "main" resource or base path (i.e. Events, /v2/core/events) so this felt safer.  I'm not married to this approach though, if we wanted to attach fetchRelatedObject directly to the Events resource.

Reply:
I'd say lets test it out - it is one less hop to have

Reply:
Tested this morning, this works!  added a comment to explain us using the Events resource for fetchRelatedObject but I think this is resolved.